addons/binding/org.openhab.binding.honeywellwifithermostat/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry exported="true" kind="lib" path="lib/json-20141113.jar"/>
```

Comment:
openHAB 2 comes with GSON as the default JSON parser. As you use this lib only at a single place in the code: Could you remove it and use GSON instead?